### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RishiHolani**
First program to simply use printk.

### CODE REVIEW

File: LKM/hny.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    return 0;   
+} 
+
+static void __exit hny_cleanup(void)
+{
+    printk(KERN_INFO "-From Rishi.\n");
+}
+
+module_init(hny_init);
+module_exit(hny_cleanup);
```
Comment (gmjain) [inline LKM/hny.c:20]: You should end every file with a newline.

File: LKM/Makefile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+obj-m += hny.o
+
+all:
+	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
+
+clean:
+	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
```
Comment (gmjain) [inline LKM/Makefile:7]: You should end every file with a newline.

File: LKM/hny.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#include <linux/module.h>    
+#include <linux/kernel.h>    
+#include <linux/init.h>      
+
+MODULE_LICENSE("GPL");
```
Comment (gmjain) [inline LKM/hny.c:5]: Where is the kernel module parameter?

Comment (RishiHolani) [reply LKM/hny.c:5]: This is the first program.

**Pull request merged**